frontend/src/components/RepresentativeTable.module.scss

```diff
@@ -0,0 +1,5 @@
+@import '../shared.scss';
+
+.memberIcon path {
+  fill: $COLOR_BLACK40;
+}
```

Comment:
This is now repeated in quite a few places, maybe it would be time to make a component for the member icon